Fix urlFinder param name to match variable in constructor. (SCAFA-1191)

diff --git a/Model/Export.php b/Model/Export.php
index 2d315a9..fba05b3 100644
--- a/Model/Export.php
+++ b/Model/Export.php
@@ -171,7 +171,10 @@ class Export
     /**
      * @var UrlFinderInterface
      */
-    protected $urlFinder;
+    //PATCH START
+    //protected $urlFinder;
+    protected $_urlFinder;
+    //PATCH END

     /**
      * @var
